redis-persistence/src/main/java/com/netflix/conductor/dao/index/ElasticSearchDAO.java

```diff
@@ -118,9 +121,16 @@
 	private static final TimeZone gmt = TimeZone.getTimeZone("GMT");
 	    
     private static final SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMWW");
+
```

Comment:
`ElasticSearchDAO` is a @singleton, whats the need of the static initialization.

Reply:
I overlooked the fact that `ElasticSearchDAO` was a singleton class. I'll make this a member variable. Thanks!

Reply:
@pctreddy Done.

Reply:
Initializing instance variable `executorService` in a static initialization block is not a good idea, can you move the initialization of `executorService` into the constructor

Reply:
@pctreddy , initialization is in the constructor.

Reply:
Missed it in the collapsed github pull request view.